hm-service-account-api-rest-v1.yml

```diff
@@ -294,6 +294,43 @@ paths:
           description: A Service Account Token associated to a fleet application
       operationId: ApiWeb.FleetImportController.index
       description: Get the status of VINs that have previously been registered for data access clearance.
+  /v1/fleets/{vin}/cancel_activation:
+    post:
+      security:
+        - ServiceAccountToken: []
+      tags:
+        - Fleet
+      responses:
+        '200':
+          schema:
+            $ref: '#/definitions/FleetVin'
+          description: Vehicle will be canceled soon
+        '422':
+          schema:
+            $ref: '#/definitions/FleetErrors'
+          description: Invalid Input. It occurs when the payload is missing a params or the the application is not configured.
+        '401':
+          schema:
+            $ref: '#/definitions/UnauthorizedErrors'
+          description: When an invalid ServiceAccountToken is used.
+        '403':
+          schema:
+            $ref: '#/definitions/FleetFrobidenErrors'
+          description: When the application doesn't have fleet access.
+        '404':
+          schema:
+            $ref: '#/definitions/FleetErrors'
+          description: Given resource not found
+        '500':
+          description: Server Errors
+      parameters:
+        - type: string
+          required: true
+          name: vin
+          in: path
+          description: Vehicle identification number
+      operationId: ApiWeb.FleetCancelActivationController.create
+      description: Enqueue fleet vehicle cancelation
```

Comment:
Description propsal: "Cancel the ongoing activation process of a vehicle that has the "pending" status. Note that this endpoint does not revoke vehicles that are in the "approved" state, for which the OAuth2 API has to be used."

Other than that looks good! Please ping me once it's live and I'll make sure to update our docs!

Reply:
Done